android/settings.gradle

```diff
@@ -11,6 +11,8 @@ include ':react-native-config'
 project(':react-native-config').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-config/android')
 include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
+include ':react-native-dev-menu'
```

Comment:
This is part of the [manual linking steps](https://github.com/zoontek/react-native-dev-menu#manual-linking). The library didn't work without this correct?

Reply:
It's the first step. It didn't work without it and it didin't work with all steps done either.